### PR TITLE
Finish the documentation required by the repo merge

### PR DESCRIPTION
**Pull request by TheLostLambda**

### CODE REVIEW

**Comment by TheLostLambda** [comment]
Ready for @ns-rse and @bobturneruk to take a look at! It would be good if you could test the contributing instructions and see if you can get the Svelte application up and running!

**Comment by TheLostLambda** [comment]
I'm happy ignoring that Codacy check, since the README was already without indents for the list items — I just happened to reword something on the same line.

File: lib/docs/contributing.md [hunk 1]
```diff
@@ -7,40 +7,54 @@ There are a few ways to contribute:
...
+### Linting (Also Used in `web`)
 
 PGFinder uses [pre-commit](https://pre-commit.com) hooks to ensure code conforms to the [PEP8 Python Style
 Guide](https://pep8.org/) using the [ruff](https://duckduckgo.com/?q=ruff+linter&t=opera&ia=images) linter, applies
-[black](https://black.readthedocs.io/en/stable/index.html) formatting and ensures Notebooks are clean on
-submission. These hooks are run on GitHub when Pull Requests are made using [pre-commit.ci](https://pre-commit.ci) and
-if you have not run them locally then your pull request will fail the tests it needs to pass.
+[black](https://black.readthedocs.io/en/stable/index.html) formatting, and checks for a number of other common mistakes.
+These hooks are run on GitHub when Pull Requests are made using [pre-commit.ci](https://pre-commit.ci), and
+if you have not run them locally, then your pull request will fail the tests it needs to pass.
```
Comment (ns-rse) [inline lib/docs/contributing.md:57]: Minor but its not guaranteed that the tests on CI will fail.

```suggestion
if you have not run them locally, then your pull request may fail the tests it needs to pass.
```

**Review by ns-rse — CHANGES_REQUESTED** [review]
Good work going through all these tweaks and updates and addition of section on the web-framework contributions is really useful (for me too!).

One minor suggestion on wording as not all `pre-commit` tests will fail.

**Review by ns-rse — APPROVED** [review]
Thanks for addressing so quickly, looks good to go.

**Pull request merged**